pages/dashboard.php

```diff
@@ -2,6 +2,39 @@
 
 use \RedBeanPHP\R as R;
 
+function getThumb($body) : string {
+  $matches = array();
+  $imgur_pattern = '`^http[s]?://i\.imgur\.com/([[:alnum:]]{7})\.(\w+)$`i';
```

Comment:
Since backticks are also the way to execute system commands in PHP, it would be ever so slightly safer to use a different delimiter. For example:
```suggestion
  $imgur_pattern = '#^http[s]?://i\.imgur\.com/([[:alnum:]]{7})\.(\w+)$#i';
```
Similar for the other regex patterns.

Reply:
Ah, sure, thanks, works for me